### PR TITLE
LPD-28178 [green ci] Delete ClientExtensionFDSFilter.testcase

### PR DESCRIPTION
**Pull request by chestofwonders**
Issue: https://liferay.atlassian.net/browse/LPD-28178

<h1>Delete ClientExtensionFDSFilter.testcase</h1>

These tests have been migrated, the remaining ones have been removed:

LPS-190220. Verify that when the user clicks on Cancel, the changes are not saved.
LPS-190220. Verify that when the user fills all fields and clicks on Publish, the changes are saved.
LPS-190220. Verify that the user can Delete the FDS Filter entry
LPS-190220. Verify that the user can edit the CX Filter entry
LPS-190220. Verify that the JavaScript URL field is mandatory
LPS-190220. Verify that the Name field cannot be empty

### CODE REVIEW

**Comment by chestofwonders** [comment]
@dsanz @markocikos @juanjofgliferay I was not able to test the name internationalization button, as it was impossible to open it from PW. I have found that it is not the same one we use in the DataSet, so I understand that we would have to ask another team for a refactoring. Am I right?

**Comment by juanjofgliferay** [comment]
@chestofwonders AFAIK, the CX page does not use Clay components (https://github.com/liferay/liferay-portal/blob/master/modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/edit_client_extension_entry.jsp#L64) as we do in the DataSet page. The behavior of the button is completely different from the one we are using in terms of accessibility (button lacks `aria-controls` and many others).
Also, the dropdown component with the languages is positioned in a different way.

**Comment by juanjofgliferay** [comment]
@chestofwonders using the `clickAndExpectToBeVisible` could do the trick :wink: 

```
	await clickAndExpectToBeVisible({
		autoClick: true,
		target: page.locator('.input-localized-palette-container'),
		trigger:
			clientExtensionsPage.newCustomElementFormModal
				.localizedNameButton,
	});
```

image

**Comment by juanjofgliferay** [comment]
Overall, PR LGTM but I was wondering why you are creating those tests in the `customElement.spec.ts` file instead of using a generic `clientExtensionAdmin.spec.ts` one or a specific `fdsFilter.spec.ts` one.

**Comment by chestofwonders** [comment]
> @chestofwonders using the `clickAndExpectToBeVisible` could do the trick 😉
> 
> ```
> 	await clickAndExpectToBeVisible({
> 		autoClick: true,
> 		target: page.locator('.input-localized-palette-container'),
> 		trigger:
> 			clientExtensionsPage.newCustomElementFormModal
> 				.localizedNameButton,
> 	});
> ```
> 
> image

Yes, it does. Love you!

**Comment by chestofwonders** [comment]
> Overall, PR LGTM but I was wondering why you are creating those tests in the `customElement.spec.ts` file instead of using a generic `clientExtensionAdmin.spec.ts` one or a specific `fdsFilter.spec.ts` one.

I thought they were better packaged into customElement.spec.ts because the differences between CX types makes a generic admin test less usable than the individual types 😗

**Comment by juanjofgliferay** [comment]
> > Overall, PR LGTM but I was wondering why you are creating those tests in the `customElement.spec.ts` file instead of using a generic `clientExtensionAdmin.spec.ts` one or a specific `fdsFilter.spec.ts` one.
> 
> I thought they were better packaged into customElement.spec.ts because the differences between CX types makes a generic admin test less usable than the individual types 😗

To solve this, we could use the same pattern used in the [FiltersPage](https://github.com/liferay/liferay-portal/blob/master/modules/test/playwright/tests/frontend-data-set-admin-web/tests/data-set-admin/pages/data_set/tabs/FiltersPage.ts#L29) where each different form extends from a shared one. No need to create all in one shot.

**Comment by dsanz** [comment]
Yep that makes sense. @chestofwonders pls check if you can do this now, in a separate subtask or in a different task we'll plan later. This may affect tests from other teams too as the `client-extension-web` pw folder contains many

**Comment by chestofwonders** [comment]
ci:test:sf

**Comment by chestofwonders** [comment]
> Yep that makes sense. @chestofwonders pls check if you can do this now, in a separate subtask or in a different task we'll plan later. This may affect tests from other teams too as the `client-extension-web` pw folder contains many

I've rather leave that for another ticket so as not to delay this PR any further.
 
I have updated it with the test to check the internationalization and I have added some fixes for the validations, since it did not always check the required fields correctly. So, please @dsanz @juanjofgliferay @markocikos check again if you see any complains to forward it

File: modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts [hunk 1]
```diff
@@ -31,6 +63,37 @@ export class ClientExtensionsPage {
...
+				'HTML Element Name Required',
+				{exact: true}
+			),
+			instanceableCheck: page.getByLabel('Instanceable', {exact: true}),
+			javaScriptUrlInput: page.getByLabel('JavaScript URL Required', {
+				exact: true,
+			}),
+			localizedNameButton: page
+				.locator('.input-localized')
+				.getByRole('button'), // page.getByRole('button', { name: 'Current translation is' }),
```
Comment (juanjofgliferay) [inline modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts:87]: :broom:

File: modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/derived-config.txt [hunk 2]
```diff
@@ -0,0 +1,586 @@
+com/liferay/frontend/taglib/clay/sample/web/internal/display/context/ClaySampleImageCard.lambda$getLabels$7(Lcom/liferay/frontend/taglib/clay/servlet/taglib/util/LabelItem;)V
```
Comment (juanjofgliferay) [inline modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/derived-config.txt:1]: @chestofwonders Curious abouth what is the purpose of this file?

Comment (chestofwonders) [reply modules/apps/frontend-taglib/frontend-taglib-clay-sample-web/derived-config.txt:1]: Commited by mistake. It's generated by gradle

File: modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts (hunk 1 above)
Comment (chestofwonders) [reply modules/test/playwright/tests/client-extension-web/pages/ClientExtensionsPage.ts]: removed!

**Review by juanjofgliferay — APPROVED** [review]
LGTM

**Comment by chestofwonders** [comment]
ci:forward

**Pull request closed without merging**